apps/transport/lib/jobs/gtfs_to_geojson_converter_job.ex

```diff
@@ -37,4 +39,10 @@ defmodule Transport.GtfsToGeojsonConverter do
       {:error, e} -> {:error, e}
     end
   end
+
+  @impl true
+  def converter, do: "rust-transit/gtfs-to-geojson"
+
+  @impl true
+  def converter_version, do: "9ca9a25b895ba1b2fdf4d04e92895afec52d0608"
```

Comment:
Je rebondis là dessus: on ne va pas utiliser ce commit comme état pour gérer l'idempotence je pense. Sinon il faut faire attention à ne pas se remettre à tout convertir par défaut pour un changement peut-être plus mineur qu'un vrai bump de version.

Je me suis demandé si on ne pourrait pas à la place utiliser https://github.com/rust-transit/gtfs-to-geojson/blob/9ca9a25b895ba1b2fdf4d04e92895afec52d0608/Cargo.toml#L3 (mais il faudra demander à ce que la version évolue lorsqu'on a une release, et on compile sur master de mémoire).

Reply:
> Je rebondis là dessus: on ne va pas utiliser ce commit comme état pour gérer l'idempotence je pense

Non je ne pense pas.

Vu l'état de maintenance/rythme d'évolutions de convertisseur j'étais parti sur un commit plutôt qu'un numéro de version. C'est plus du "best effort" qu'un versioning auquel je crois vraiment.

Reply:
Ah mais en fait j'avais mélangé commit SHA dans la migration mais mis le numéro de version dans le job de conversion, je passe en numéro de version partout 👍 Merci ! https://github.com/etalab/transport-site/pull/3549/commits/02fa6120110dcc67c69e6109c070804addd27526

Reply:
Ok ça marche ! Merci